docs/contributing.md

```diff
@@ -10,24 +10,21 @@ has_children: false
 
 Thank you for your interest in contributing to our project. Whether it's a bug report, new feature, correction, or additional documentation, we greatly value feedback and contributions from our community.
 
-Please read through this document before submitting any issues or pull requests to ensure we have all the necessary information to effectively respond to your bug report or contribution..
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary information to effectively respond to your bug report or contribution.
 
-If you don't know where to start, please [join our community on Slack](https://lakefs.io/slack) and ask us. We will help you get started!
+Don't know where to start? [Join our community on Slack](https://lakefs.io/slack) and we will help you get started. We also recommend this [free series](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github){:target="_blank"} about contributing to OSS projects.
+{: .note .note-info }
 
-## Ground Rules
+## Getting Started
 
 Before you get started, we ask that you:
 
 * Check out the [code of conduct](https://github.com/treeverse/lakeFS/blob/master/CODE_OF_CONDUCT.md).
 * Sign the [lakeFS CLA](https://cla-assistant.io/treeverse/lakeFS) when making your first pull request (individual / corporate)
-* Submit any security issues directly to [security@treeverse.io](mailto:security@treeverse.io)
-
-
-## Getting Started
-
-Want to report a bug or request a feature? Please [open an issue](https://github.com/treeverse/lakeFS/issues/new)
```

Comment:
Think we should leave this or something similar

Reply:
It was replaced by the 4th bullet on the Getting started list.